wordpress-mobile/gutenberg-mobile
# Remove auto-generated files from bundle and add to .gitignore

Fixes #2693

This change removes autogenerated files in the bundle directory from source control.

WPAndroid Test PR:https://github.com/wordpress-mobile/WordPress-Android/pull/13068
WPiOS Test PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15060

To test:
WPAndroid
1. Run a local WPAndroid build and make sure everything works as expected
2. Test Automated WPAndroid build and make sure everything works as expected

WPiOS
1. Run a local  WPiOS build and make sure everything works as expected
2. Test Automated WPiOS build and make sure everything works as expected

DemoApp
3. Run Demo app from gutenberg-mobile on Android and iOS and make sure everything works as expected
.4 Run demo app from gutenberg on Android and iOS and make sure everything works as expected. 

PR submission checklist:
